projects/chaos/inc/chaos_flags.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+// Flags for Chaos to disable/enable commonly toggled features.
+
+#define CHAOS_FLAG_ENABLE_SOLAR_RELAYS 1
```

Comment:
Probably don't want to define this as `1` if you're using `#ifdef` - I think `0` would still count as defined?